pyartcd/pyartcd/util.py

```
@@ -82,13 +82,15 @@ async def load_group_config(
     env=None,
     doozer_data_path: str = constants.OCP_BUILD_DATA_URL,
     doozer_data_gitref: str = '',
+    extra_vars: List = [],
```

Ruff (0.15.21)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)